docs/enterprise-guide/setting-up-sso.md

```diff
@@ -0,0 +1,144 @@
+## Let you customers log in with SSO using SAML or JWT
+The Enterprise edition of Metabase lets you connect your SAML- or JWT-based SSO to it. Integrating your SSO with Metabase allows you to:
+
+* automatically pass user attributes from your SSO to Metabase in order to power data sandboxes
+* let your users access Metabase without re-authenticating.
+
+There are slightly different steps to take depending on whether your SSO solution uses SAML or JWT. We'll cover SAML first.
+
```

Comment:
we should include a link to https://metabase.com/docs/latest/administration-guide/10-single-sign-on.html and note that these options exist.